net/hs20/Makefile

```diff
@@ -1,15 +1,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=hs20
-PKG_RELEASE:=4
+PKG_RELEASE:=5
 
 PKG_SOURCE_URL:=http://w1.fi/hostap.git
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_DATE:=2020-06-08
 PKG_SOURCE_VERSION:=5a8b366233f5585e68a4ffbb604fbb4a848eb325
 
 PKG_SOURCE_NAME:=hostapd
-PKG_MIRROR_HASH:=1b2a4947034142587a3f81eac2ea27dd64650e77f8943d973623b47533be2393
+PKG_MIRROR_HASH:=243af130db0310a00b14ee5929d858264b26df321c47e509dbfe12d6b99e6e3b
 PKG_MIRROR_VERSION:=$(PKG_SOURCE_DATE)-$(call version_abbrev,$(PKG_SOURCE_VERSION))
 PKG_SOURCE:=$(PKG_SOURCE_NAME)-$(PKG_MIRROR_VERSION).tar.xz
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)/$(PKG_SOURCE_NAME)-$(PKG_MIRROR_VERSION)
```

Comment:
PKG_SOURCE_NAME to PKG_NAME

Reply:
Actually this should just be cleaned up. There's no good reason for all this hackery.